index.bs

```diff
@@ -124,34 +124,35 @@ Issue: We should consider having further normative restrictions on file names th
 never be allowed using this API, rather than leaving it entirely up to underlying file
 systems.
 
+A <dfn for="file entry">lock type</dfn> is a [=string=] that may exclusively be "`open`",
```

Comment:
`for="blah object"` is for defining members of "blah object", whereas in this case we're basically defining a new enum type. I would recommend defining it like e.g., [`storage type`](https://github.com/whatwg/storage/blob/7d89528a55e1730284755870f326c9bd37ef50e6/storage.bs#L164)